parcels/kernel.py

```diff
@@ -4,9 +4,12 @@
 from parcels.field import FieldSamplingError
 from parcels.loggers import logger
 from parcels.kernels.advection import AdvectionRK4_3D
-from os import path
+from os import path, remove
 import numpy.ctypeslib as npct
+import time
 from ctypes import c_int, c_float, c_double, c_void_p, byref
+import _ctypes
```

Comment:
What's the reason for now importing all classes of `_ctypes`? Which ones were we missing before (`c_int, c_float, c_double, c_void_p, byref`)? Can't we just add the extra classes required by Windows (`FreeLibrary`, `dlclose`)?